cmd/zt_copy_s2smigration_test.go

```diff
@@ -247,6 +306,47 @@ func (s *cmdIntegrationSuite) TestS2SCopyFromS3ToBlobWithWildcardInSrcAndBucketN
 	})
 }
 
+func (s *cmdIntegrationSuite) TestS2SCopyFromGCPToBlobWithWildcardInSrcAndBucketNameNeedBeResolved(c *chk.C) {
+	skipIfS3Disabled(c)
```

Comment:
we can perhaps rename skipIfS3Disabled since it controls both S3 and GCP tests

Reply:
What if we specialize for GCP so that way we can control S3 tests and GCP individually?

Reply:
Separating GCP and S3 tests